src/test/java/com/wikia/webdriver/common/core/imageutilities/SimpleViewportRelativeCoordsProvider.java

```diff
@@ -0,0 +1,48 @@
+package com.wikia.webdriver.common.core.imageutilities;
+
+import com.wikia.webdriver.common.driverprovider.DriverProvider;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import ru.yandex.qatools.ashot.coordinates.Coords;
+import ru.yandex.qatools.ashot.coordinates.CoordsProvider;
+
+import java.awt.*;
+import java.util.ArrayList;
+
+public class SimpleViewportRelativeCoordsProvider extends CoordsProvider {
+
+    private WebDriver driver = DriverProvider.getActiveDriver();
+    private JavascriptExecutor js = (JavascriptExecutor) driver;
+
+    @Override
+    public Coords ofElement(WebDriver driver, WebElement element) {
+
+        Dimension dimension = element.getSize();
+        Point start = (Point) getBoundingClientRect(element, driver)[0];
+
+        Coords coords =  new Coords(
+                start.x,
+                start.y,
+                dimension.getWidth(),
+                dimension.getHeight());
+
+        PageObjectLogging.log("SimpleViewportRelativeCoordsProvider",
+                "Start: " + coords.x + "x" + coords.y + " , size: " + coords.width + "x" + coords.height, true);
+        return coords;
+    }
+
+    private Object[] getBoundingClientRect(WebElement element, WebDriver driver) {
+        JavascriptExecutor js = (JavascriptExecutor) driver;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "js" which hides the field declared at line 18. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AHiddenFieldCheck)